OADP 3400 storage class mapping (#72284)
openshift/openshift-docs: modules/oadp-storage-class-mapping-oadp.adoc

// Module included in the following assemblies:
//
// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc

[id="oadp-storage-class-mapping-oadp_{context}"]
:_mod-docs-content-type: PROCEDURE

[id=storage-class-mapping-oadp_{context}]

> = Storage class mapping with OADP

Reviewer: Proc should have a gerund title, does this work..?
```suggestion
= Mapping storage classes with OADP
```

Author: Thank you!!!

> You can use OpenShift API for Data Protection (OADP)  with the Velero plugin v1.1.0 and above to change the storage class of a persistent volume (PV) during restores, by configuring a storage class mapping in the config map in the Velero namespace.

Reviewer: Per ISG
```suggestion
You can use OpenShift API for Data Protection (OADP)  with the Velero plugin v1.1.0 and later to change the storage class of a persistent volume (PV) during restores, by configuring a storage class mapping in the config map in the Velero namespace.
```

> To deploy ConfigMap with OADP, use the `change-storage-class-config` field. Ensure to change the storage class mapping based on your cloud provider.

Reviewer: ```suggestion
To deploy ConfigMap with OADP, use the `change-storage-class-config` field. You must change the storage class mapping based on your cloud provider.
```

> 

Reviewer: ```suggestion

.Procdure

```
